src/ldclient_config.erl

```diff
@@ -395,13 +406,15 @@ get_all() ->
     {ok, Instances} = application:get_env(ldclient, instances),
     Instances.
 
--spec tls_base_options() -> [ssl:tls_client_option()].
-tls_base_options() ->
-    DefaultCipherSuites = ssl:cipher_suites(default, 'tlsv1.2'),
-    CipherSuites = ssl:filter_cipher_suites(DefaultCipherSuites, [
+-spec get_suites(TlsVersion :: ssl:protocol_version()) -> ssl:ciphers().
+get_suites(TlsVersion) ->
+    DefaultCipherSuites = ssl:cipher_suites(default, TlsVersion),
```

Comment:
The cipher_suites function gives you all ciphers supported by the specified version and previous versions. `default` are the default suites, so some are already disabled for security. Not enough to pass my existing TLS tests. 

`exclusive` gives you the ones exclusive to that version. I considered adding together the ones we had used for 1.2, and the ones that are exclusive to 1.3. That did work, but this seems a little cleaner.